collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -171,16 +171,7 @@ public void setExternalDataManager(ExternalDataManager externalDataManager) {
     }
 
     public String getVersionedAppName() {
-        String versionName = "";
-        try {
-            versionName = getPackageManager()
-                    .getPackageInfo(getPackageName(), 0)
-                    .versionName;
-            versionName = " " + versionName.replaceFirst("-", "\n");
```

Comment:
Hi, Even after deleting the .git folder this solution works. I've tried and tested the same.